src/commands/poll.ts

```diff
@@ -15,39 +16,47 @@ function getOptionsString(options: any) {
 
 async function cmdPoll(parsed: ParsedMessage, client: Client) {
   // ?poll target "Question" emoji1 "response meaning" emoji2 "xxxxx"...
-  const args = parsed.arguments;
-  const target = parsed.arguments[0];
+  try {
```

Comment:
The try except should just go in the main since it is the same for every command